spec/actors/hyrax/actors/lease_actor_spec.rb

```diff
@@ -35,4 +35,24 @@
       end
     end
   end
+
+  context 'deactivating an expired lease', clean_repo: true do
+    let(:lease_attributes) do
+      { lease_date: Date.tomorrow.to_s,
+        current_state: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_PUBLIC,
+        future_state: Hydra::AccessControls::AccessRight::VISIBILITY_TEXT_VALUE_AUTHENTICATED }
+    end
+    let(:leased_work) { create(:leased_work, with_lease_attributes: lease_attributes) }
+    let(:subject) { described_class.new(leased_work) }
+
+    before do
+      leased_work
```

Comment:
I don't think you need this since `leased_work` will be called first thing on line 53.  If that wasn't the case, you could make line 45 be `let!` which would force the loading of `leased_work`.